frontend/src/app/shared/components/header/header.component.sass

```diff
@@ -27,6 +27,19 @@
 :host ::ng-deep .change-color
   background-color: $cornflower
 
+::ng-deep .modal-content
```

Comment:
```suggestion
:host ::ng-deep .modal-content
```

Reply:
Also, I don't see using of modal-content class

Reply:
Adding host disables that style and modal-content is needed to control the whole modal's background

Reply:
but without :host this can affect not only one component

Reply:
I added a class to the opening modal so styles will target a particular one